packages/xrpl/test/integration/utils.ts

```diff
@@ -78,29 +231,64 @@ export async function verifySubmittedTransaction(
   }
 }
 
+/**
+ * Sends a test transaction for integration testing.
+ *
+ * @param client - The XRPL client
+ * @param transaction - The transaction object to send.
+ * @param wallet - The wallet to send the transaction from.
+ * @param retry - As of Sep 2022, xrpl.js does not track requests sent in parallel. Our sequence numbers can get off from
+ *               the server's sequence numbers. This is a fix to retry the transaction if it fails due to tefPAST_SEQ.
+ * @param retry.count - How many times the request should be retried.
+ * @param retry.delayMs - How long to wait between retries.
+ * @returns The response of the transaction.
+ */
+// eslint-disable-next-line max-params -- Test function, many params are needed
 export async function testTransaction(
```

Comment:
This should go in a quick followup (I don't want to re-review files rn), but perhaps this method should be renamed `testTransactionSuccess`, since it's checking for success.

Reply:
Added to followup